lua/rose-pine.lua

```diff
@@ -923,6 +923,24 @@ local function set_highlights()
 		RainbowDelimiterRed = { fg = palette.love },
 		RainbowDelimiterViolet = { fg = palette.iris },
 		RainbowDelimiterYellow = { fg = palette.gold },
+
+		-- MeanderingProgrammer/markdown.nvim
+		RenderMarkdownBullet = { fg = palette.rose },
+		RenderMarkdownChecked = { fg = palette.foam },
+		RenderMarkdownCode = { bg = palette.overlay },
+		RenderMarkdownCodeInline = { fg = palette.text, bg = palette.overlay },
+		RenderMarkdownDash = { fg = palette.muted },
+		RenderMarkdownH1Bg = { bg = palette.iris, blend = 20 },
+		RenderMarkdownH2Bg = { bg = palette.foam, blend = 20 },
+		RenderMarkdownH3Bg = { bg = palette.rose, blend = 20 },
+		RenderMarkdownH4Bg = { bg = palette.gold, blend = 20 },
+		RenderMarkdownH5Bg = { bg = palette.pine, blend = 20 },
+		RenderMarkdownH6Bg = { bg = palette.leaf, blend = 20 },
```

Comment:
We should use `groups.<h1...h6>` here. We can also update `groups.h6` to "leaf" if necessary.